tests/scripts/all.sh

```diff
@@ -2114,6 +2115,7 @@ component_test_valgrind_constant_flow () {
     scripts/config.py full
     scripts/config.py set MBEDTLS_TEST_CONSTANT_FLOW_VALGRIND
     scripts/config.py unset MBEDTLS_USE_PSA_CRYPTO
+    scripts/config.py --force set MBEDTLS_VALGRIND
```

Comment:
Why is there a single option and not two, one for memory and one for constant-time? The set of tests that are of high interest is different for those two.

Reply:
That's true, but none of the constant-flow tests actually have any tests that are conditional on Valgrind (given that I have made the `constant_time_hmac` changes unconditional). So I have left this set for now, for consistency.